LBH.AdultSocialCare.Api/V1/Controllers/Common/CarePackageReclaimController.cs

```diff
@@ -152,5 +160,38 @@ public async Task<ActionResult<decimal>> GetProvisionalCareChargeAmountUsingServ
             var provisionalAmount = await _getCareChargeUseCase.GetUsingServiceUserIdAsync(serviceUserId);
             return Ok(provisionalAmount.Amount);
         }
+
+        /// <summary>
+        /// Gets the paginated care package list with care charge information.
+        /// </summary>
+        /// <param name="parameters">Parameters to filter list of care packages.</param>
+        /// <returns>List of care packages with care charge status</returns>
+        [HttpGet("care-charges/packages")]
+        [ProducesResponseType(typeof(PagedCareChargePackagesResponse), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(string), StatusCodes.Status400BadRequest)]
+        [ProducesDefaultResponseType]
+        public async Task<ActionResult<PagedCareChargePackagesResponse>> GetCareChargePackages([FromQuery] CareChargePackagesParameters parameters)
+        {
```

Comment:
Consider using generic LBH.AdultSocialCare.Api.V1.Boundary.Common.Response.PagedResponse<T> to reduce number of PagedXXXResponse objects